fission-cli/library/Fission/CLI/Release.hs

```diff
@@ -0,0 +1,24 @@
+module Fission.CLI.Release (checkLatestRelease) where
+
+import           Fission.Prelude
+
+import           GitHub (github')
+import qualified GitHub
+import           GitHub.Data.Releases                           as Releases
+
+checkLatestVersion ::
+  ( MonadLogger m
+  , MonadIO     m
+  )
+  => m ()
+checkLatestRelease = do
+  -- check "last checked" timestamp to see if we need to check again
+  logInfo @Text "Checking for newer versions..."
+
+  possibleVersion <- liftIO $ github' GitHub.latestReleaseR "fission-suite" "fission"
+  case possibleVersion of
+    Left  _ -> return () -- just ignore errors here.
+    Right latestVersion -> do
+      -- check latest version against current version from Meta.package
+      logInfo $ "latest version is: " <> Releases.releaseTagName latestVersion
+      return ()
```

Comment:
NO NEED TO CHANGE, but fun fact: 

```haskell
logInfo :: MonadLogger m => Text -> m ()
```

So the resulting type of `logInfo` is `m ()`, which is the same as `return ()`. You can remove line 24 and it'll be the same.

I realize that the current code is WIP and line 24 is probably from when you wrote the initial branching. Also nothing wrong with leaving it in there (it'll have zero runtime cost). Just using it to highlight how to think in Haskell :)

Reply:
Oh, also before I forget: possibly another helpful tip...

If you need to just stub out a function or value as a "I'll look at that later", using the value `undefined` will fill literally any type. If the program executes that instruction, it'll crash, but helpful when developing as a placeholder.